spec/controllers/users_controller._pec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #show' do
+    before(:each) { get user_path(1) }
+
+    it 'returns http success' do
+      get :show
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #new' do
+    it 'returns http success' do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the new template' do
+      get :new
+      expect(response).to render_template('new')
+    end
+
+    it 'renders the correct text on template' do
+      expect(response.body).to include('User show page')
+    end
+  end
+end
```

Comment:
- [ ] These last four tests failed. Please use `type: :request` instead `type: :controller` as it a requirement for this project. Also kindly fix the failing tests.

<img width="590" alt="Code_X9mbdLCHyn" src="https://user-images.githubusercontent.com/81468675/178569207-d2bcdc73-4c37-4d08-9699-1f8a51a9dee0.png">